src/native/managed/cdacreader/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/StackWalk/FrameHandling/FrameIterator.cs

```diff
@@ -144,19 +144,13 @@ private static FrameType GetFrameType(Target target, TargetPointer frameIdentifi
     {
         foreach (FrameType frameType in Enum.GetValues<FrameType>())
```

Comment:
This isn't introduced by your change, but this loop seems expensive for something we do for every frame. You don't need to change it now but we may want to revisit